Samples.Objc/Samples.ObjC/API/Endpoints/CreateOrderEndpoint.h

```diff
@@ -1,21 +1,17 @@
 //
 //  CreateOrderEndpoint.h
-//  PayPalCheckout-Samples-iOS-Objc
+//  PayPalNativeCheckoutObjC
 //
-//  Created by Haider Khan on 5/19/20.
-//  Copyright © 2020 PayPal. All rights reserved.
+//  Created by Nguyen, The Nhat Minh on 3/14/21.
```

Comment:
Please revert this, or add the copyright header `Copyright © 2020 PayPal. All rights reserved.`